web/src/components/EventsDateFilter.tsx

```diff
@@ -60,19 +59,19 @@ const EventsDateFilter = ({
   setToDate,
   toDateError,
 }: EventsDateFilterProps): JSX.Element => {
-  const defaultFromDate = DateTime.local().startOf('day')
-  const defaultToDate = DateTime.local().plus({ day: 10 }).startOf('day')
+  const defaultFromDate = DateTime.now().startOf('day')
+  const defaultToDate = DateTime.now().plus({ day: 10 }).startOf('day')
   const [toggleDateFilter, setToggleDateFilter] = useState(true)
```

Comment:
I think I'd rather have a closed filter as default (if the default `toDate` is set reasonably).
```suggestion
  const [showDateFilter, setShowDateFilter] = useState(false)
```